recipes/falcon-auth/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "falcon-auth" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 21ee33daf8b615fb24392b6f80bc2287d02bd0a3c4a924180d2678d5cc55838d
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - pytest-runner
+  run:
+    - python
+    - falcon
+    - pyjwt
+
+test:
+  commands:
```

Comment:
Is there a reason to not use a simple `imports` test?

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-imports

I think:

```
test:
    imports:
        - falcon-auth
```

should get all of it given:

https://github.com/loanzen/falcon-auth/blob/master/falcon_auth/__init__.py

Reply:
thanks @synapticarbors, you're perfectly right! Sorry for overlooking this.